Makefile

```diff
@@ -1,7 +1,7 @@
-PREFIX ?=	$PREFIX
+include .config
```

Comment:
Confirmed to work on following variants:
- smake
- NetBSD/bmake
- FreeBSD
- Dragonfly